docs/lib/getting-started/add-api.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Add API and database
+title: Connect DataStore to API
 description: Getting Started with Amplify Libraries - how to add API and database to your app.
```

Comment:
Instead of connect datastore to API, maybe it would be better to say "Sync Datastore to the cloud"

Reply:
Hmm. I won't say no right away, but something about "the cloud" seems not credible. Need a tie breaker: @brene Can you choose the verbiage here?

![](https://i.pinimg.com/originals/12/36/5f/12365fc27db1a2820559cf24d5bd5fa4.jpg)

Reply:
Maybe
* Connect datastore to the cloud
* Connect datastore to backend

I guess i was never a fan of the "API" name because API is generic and may or may not imply 'backend'.  APIs are application programmable interfaces that may or may not be online.

Reply:
Sounds like you're arguing for Connect DataStore to a Backend API which I don't mind, modulo the length.

Reply:
Something along those lines, but not a fan of "Backend API".  I think what I'm getting at is that the first page we promise that we will "Connect your local data to synchronize to a cloud backend". So having a section called "Connect DataStore to API" doesn't (on the surface) seem like it would match up to what we initially stated.

Reply:
My vote is for: Connect DataStore to the cloud

Reply:
![](https://thumbs.gfycat.com/ActualThirstyCricket-size_restricted.gif)